CalcualtorLogic.js

```diff
@@ -14,9 +14,13 @@ class CalculatorLogic {
     };
     divide(number1, number2) {
         let result = number1 / number2;
+        if (number2 === 0) result.textContent = 'Cannot divide by zero';
```

Comment:
where from this result.textcontent came from?

Reply:
yes you are right I noticed that it was from my pocket 
so I have removed it and pushed the update before seeing your question !!